src/main/com/example009/radek/v3/FileService.java

```diff
@@ -0,0 +1,102 @@
+package main.com.example009.radek.v3;
+
+import java.io.*;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileService {
+
+    private final String PATH_FILE_RESULTS = "src/main/com/example009/radek/v3/calculator-results/";
+
+    void storeResultsIntoFile(List<Double> inputList) {
+
+        String name = getUserInput("Please insert file name where you want to save results: ");
+        saveResultsIntoFile(name, inputList);
+    }
+
+    void loadResultsFromFile() throws IOException {
+
+        try {
+            String name = getCheckedUserInput("Please insert file name where you want to load results from: ");
+            BufferedReader reader = new BufferedReader(new FileReader(PATH_FILE_RESULTS + name));
+            reader.lines().forEach(System.out::println);
+            reader.close();
```

Comment:
ten reader by se mel closnout v sekci finally ...ktera tady ani neni

Reply:
Upraveno

Reply:
spatne, pokud to spadne a neco se pokazi, tak to skoci rovnou do catch bloku a ten writer se neuzavre! musis za catch dodat block finally ... ten se vola v obou pripadech ... psal jsi o try with resources, ale nevidim to 
tady je spravne pouziti https://www.baeldung.com/java-try-with-resources

btw na tomto serveru je vzdy vse spravne a hlavne citelne popsano ... je tam snad 95% vsech problemy napsanych a podrobne vysvetlenych

Reply:
V novějším commitu by to mělo být podle té stránky, ale toto je outdated a ještě to je po staru, mrkni prosím na ten nejnovější commit před pár dny, jestli je to ok, nebo jen dej vědět, že není případně

Reply:
Pořád je to špatně, buď to máš v jiné branchi nebo to stále není správně použito, z toho co vidím, tak to není správně použito.

Reply:
už je to ok v load metode, ted jeste v te save